### PR TITLE
Standardise eth3d dataset storage location

### PR DESCRIPTION
**Pull request by TedVanderfeen**
The current `download_eth3d.sh` script downloads the eth3d data to `datasets/eth3d/train2/`. But the evaluation script (`eval_eth3d.sh`) looks for this data in `datasets/eth3d/train/`.

The eth3d dataset is the only one that has this file structure, and not just a `/datasets/xxx/` folder. So I've made changes to bring eth3d in line with how the other datasets are stored and retrieved for evaluation.

### CODE REVIEW

**Comment by rmurai0610** [comment]
Hi thank you for catching this!
But I'll rather keep the train/ there to clarify that we are running on the training set not the test set (eth3d has both train and test sequences) 
If you can change the PR to keep the `eth3d/train` directory (i.e. only remove 2 from train2) that would be great.

**Comment by TedVanderfeen** [comment]
No problem, fixed.
